app/src/components/market/sections/market_sell/market_sell.tsx

```diff
@@ -280,7 +287,7 @@ const MarketSellWrapper: React.FC<Props> = (props: Props) => {
               title={'Total'}
               value={`${
                 tradedCollateral ? formatNumber(formatBigNumber(tradedCollateral, collateral.decimals, 2)) : '0.00'
-              } ${collateral.symbol}`}
+              } ${symbol}`}
```

Comment:
We could add a dropdown similar to https://github.com/protofire/omen-exchange/pull/1570/files#diff-b9e7bcd4798ff09ab957e9c99ff18b23f2ce18875844965cf021e5600cd13c94R175
Allowing user to withdraw either in native asset or wrapped native asset.

Reply:
yeah definitely that will be awesome! will follow this up in https://github.com/protofire/omen-exchange/issues/1538